src/ShopRatingService/DataAccess/ShopRatingServiceTrustedShopsDataAccessInterface.php

```diff
@@ -13,7 +13,26 @@
 
 interface ShopRatingServiceTrustedShopsDataAccessInterface
 {
+    /**
+     * @param string $remoteKey
+     * @return int
+     */
     public function getItemCountForRemoteKey($remoteKey);
 
+    /**
+     * @param array<string, mixed> $data
+     * @psalm-param array{
+     *     insertId: string,
+     *     pkgShopRatingServiceId: string,
+     *     remoteKey: string,
+     *     score: int|'',
+     *     rawData: string,
+     *     ratingUser: string,
+     *     ratingText: string,
+     *     ratingDate: string,
+     * } $data
+     *
+     * @return void
+     */
     public function insertItem(array $data);
```

Comment:
The implementation of this method depends on the `$data` array having *exactly* this format because it is passed directly into a DBAL query.